### ISSUE TITLE
params of 3D model may be wrong

### ISSUE THREAD

**Issue by chenxiahan** [sovrasov/flops-counter.pytorch#42]
Sorry, I found another problem. When I calculate the 3D model's params, the answer is different from my own results. My own code is
```
def count_param(model):
    param_count = 0
    for param in model.parameters():
        param_count += param.view(-1).size()[0]
    return param_count
```
A 3D model's param calculated by my code is 166.21M and by your code is 160.64 M

**Comment by sovrasov** [comment]
That happens because ptflops takes into account parameters for supported modules only. Your model contains such modules and they are not contributing to the final numbers. I'll think about changing that behavior for the number of parameters.

**Comment by chenxiahan** [comment]
I got it. Thank you.

**Comment by sovrasov** [comment]
Resolved in #43
